csds-client/main.go

```diff
@@ -2,11 +2,55 @@ package main
 
 import (
 	"envoy-tools/csds-client/client"
+	"flag"
 	"log"
+	"time"
 )
 
+// ParseFlags parses flags to ClientOptions
+func ParseFlags() client.ClientOptions {
+	const uriDefault string = "trafficdirector.googleapis.com:443"
+	const platformDefault string = "gcp"
+	const authnModeDefault string = "auto"
+	const apiVersionDefault string = "v2"
+	const requestFileDefault string = ""
+	const requestYamlDefault string = ""
+	const jwtDefault string = ""
+	const configFileDefault string = ""
+	const monitorIntervalDefault time.Duration = 0
+	const visualizationDefault bool = false
```

Comment:
To be honest I see little value in defining default parameters as const, as it is equivalent of defining them inline with flags. I understand this was a suggestion by another reviewer, I'll leave it up to your judgement in case I am missing the value of the approach.